### DISCUSSION TITLE
NON RADIO SUPPORT

### DISCUSSION THREAD

**Discussion by kalyan10** [srsran/srsRAN_Project#38] [General Help] [answered]
whether  srsranproject support zmq mode or not?
if it is supporting give the details to execute the same.

which UE it is suporting? srs ue is helpful or not?

**Comment by herlesupreeth** [comment]
Please refer to the discussion here - https://github.com/srsran/srsRAN_Project/issues/32

**Comment by brendan-mcauliffe** [comment, marked as answer]
You can now find a tutorial showing how to use the srsRAN Project gNB with srsUE over ZMQ (and over-the-air) [here](https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html).
